packages/webdriverio/src/commands/browser/mock.ts

```diff
@@ -4,8 +4,10 @@ import WebDriverNetworkInterception from '../../utils/interception/webdriver.js'
 import { getBrowserObject } from '../../utils/index.js'
 import type { Mock } from '../../types.js'
 import type { MockFilterOptions } from '../../utils/interception/types.js'
+import type { CDPSession } from 'puppeteer-core/lib/esm/puppeteer/common/Connection.js'
 
 export const SESSION_MOCKS: Record<string, Set<Interception>> = {}
+export const CDP_SESSIONS: Record<string, CDPSession> = {}
```

Comment:
`const client = await page.target().createCDPSession()` creates new CDPSession, but we need the original one to disable fetch domain. Storing `map<handle, CDPSession>` allows us to restore mocks in any order